docs/kane-ai-smart-variables.md

```diff
@@ -0,0 +1,134 @@
+|                       | `end_of_month`        | `{{smart.end_of_month}}`    | Date of the last day of the current month (YYYY-MM-DD)                      |
+| **User/System Information** | `os_type`             | `{{smart.os_type}}`       | Operating system name (e.g., Windows, macOS, Linux)                          |
+|                       | `os_version`          | `{{smart.os_version}}`    | Operating system version (e.g., 10.0, 14.2)                                 |
+|                       | `browser_name`        | `{{smart.browser_name}}`    | Browser name (e.g., Chrome, Firefox, Safari)                                |
+|                       | `browser_version`     | `{{smart.browser_version}}` | Browser version (e.g., 112.0, 101.0.1)                                      |
+|                       | `user_name`           | `{{smart.user_name}}`     | Current user name on LambdaTest who triggered the test                       |
+| **Geo/Location** | `country`             | `{{smart.country}}`       | Current country name (e.g., United States, India)                           |
+|                       | `city`                | `{{smart.city}}`          | Current city name (e.g., New York, Delhi)                                   |
+|                       | `latitude`            | `{{smart.latitude}}`      | Current latitude                                                              |
+|                       | `longitude`           | `{{smart.longitude}}`     | Current longitude                                                             |
+| **Connectivity** | `ip_address`          | `{{smart.ip_address}}`    | Current IP address (IPv4)                                                     |
+| **Randomized** | `random_int`          | `{{smart.random_int}}`    | Random 3-digit integer                                                        |
+|                       | `random_float`        | `{{smart.random_float}}`  | Random 2-digit float                                                          |
+|                       | `random_string_8`     | `{{smart.random_string_8}}` | Random alphanumeric string of length 8                                        |
+|                       | `random_string_56`    | `{{smart.random_string_56}}`| Random alphanumeric string of length 56                                       |
+|                       | `random_email`        | `{{smart.random_email}}`  | Random email address                                                          |
+|                       | `random_phone`        | `{{smart.random_phone}}`  | Random phone number                                                           |
+| **HTTP Request/Response** | `http_status_code`    | `{{smart.http_status_code}}`| Status code of the last HTTP request (e.g., 200, 404)                        |
```

Comment:
Get this checked by Sanyam if all have been added in release. I think some are missing